### PR TITLE
How-to get snapshot versions

### PR DESCRIPTION
**Pull request by jamowei**

### CODE REVIEW

File: ProjectSetup.md [hunk 1]
```diff
@@ -90,4 +88,31 @@ fun main() {
 
 Run the project by calling `./gradlew jsRun` in your project's main directory. Add `--continuous` to enable automatic building and reloading in the browser when you change your code.
```
Comment (EvaBS) [inline ProjectSetup.md:89]: .. after changing your code?

File: ProjectSetup.md [hunk 2]
```diff
@@ -90,4 +88,31 @@ fun main() {
...
 
-And voilà, you are done! Maybe you would like to create some more versatile [HTML](Attributes%20and%20CSS.html) now?  
\ No newline at end of file
+And voilà, you are done! Maybe you would like to create some more versatile [HTML](Attributes%20and%20CSS.html) now?  
+
+## Pre-release builds
+If you want get the newest snapshot-builds of fritz2 before we are releasing them you can put the following into your 'build.gradle.kts':
```
Comment (EvaBS) [inline ProjectSetup.md:94]: ..before we release them, put the following...

File: ProjectSetup.md [hunk 3]
```diff
@@ -90,4 +88,31 @@ fun main() {
...
+            dependencies {
+                implementation(kotlin("stdlib"))
+                implementation("dev.fritz2:core:0.7-SNAPSHOT") // add the newer snapshot version here
+            }
+        }
+        ...
+    }
+}
+```
+If you find any bugs or problems when using this snapshots-versions please give us an issue on
```
Comment (EvaBS) [inline ProjectSetup.md:117]: ..problems with these snapshots-versions, please open an issue..

**Comment by jamowei** [comment]
@EvaBS it would be much easier when you do these corrections directly in the source files and push a new commit ;-)

**Pull request merged**